eclipse-ee4j/jaxrs-api
# TCK Migration: migrate last set of tests from ee/rs

This is to migrate the last set of tests from jakartaee-tck jaxrs/ee/rs/ folder 

To test the changes :

export JAVA_HOME=<JDK11_HOME>
export M2_HOME=<MAVEN_HOME>
export PATH=$M2_HOME/bin:$JAVA_HOME/bin:$PATH

jaxrs-tck/
mvn clean install

jersey-tck/
mvn clean verify -Parq-glassfish-managed -Dit.test=jakarta.ws.rs.tck.ee.rs.**

The tests migrated :
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/formparam
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/get
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/head
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/headerparam
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/matrixparam
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/options
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/pathparam
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/produceconsume
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/put
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/queryparam

All tests should pass.

This PR will be ready for review post merging of #1031

Signed-off-by: alwin-joseph <alwin.joseph@oracle.com>
